ci-operator/step-registry/upi/deprovision/azurestack/upi-deprovision-azurestack-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+TENANT_ID=$(cat ${SHARED_DIR}/TENANT_ID)
+APP_ID=$(cat ${SHARED_DIR}/APP_ID)
+AAD_CLIENT_SECRET=$(cat ${SHARED_DIR}/AAD_CLIENT_SECRET)
+
+AZURESTACK_ENDPOINT=$(cat ${SHARED_DIR}/AZURESTACK_ENDPOINT)
+SUFFIX_ENDPOINT=$(cat ${SHARED_DIR}/SUFFIX_ENDPOINT)
+RESOURCE_GROUP=$(cat "${SHARED_DIR}/RESOURCE_GROUP_NAME")
+
+az cloud register \
+    -n PPE \
+    --endpoint-resource-manager "${AZURESTACK_ENDPOINT}" \
+    --suffix-storage-endpoint "${SUFFIX_ENDPOINT}" 
+az cloud set -n PPE
+az cloud update --profile 2019-03-01-hybrid
+az login --service-principal -u $APP_ID -p $AAD_CLIENT_SECRET --tenant $TENANT_ID
```

Comment:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/18796/rehearse-18796-pull-ci-openshift-installer-master-e2e-azurestack-upi/1433081210522832896/artifacts/e2e-azurestack-upi/upi-deprovision-azurestack/build-log.txt
Seems to have an error: 
```
ERROR: Please run 'az login' to setup account.
```
I noticed the other invocation of this command redirects to `/dev/null` 
Should that be done here? Any other ideas what's going on here?

Reply:
It redirects to /dev/null so it doesn't display sensitive information. I removed it because I wanted to know if there is an error showing up during installation. I can't seem to understand why the next step says login when I'm doing it right before that.